dusty/commands/utils.py

```diff
@@ -1,8 +1,16 @@
+from copy import copy
 import os
 import subprocess
 
+from ..systems.docker import get_docker_env
+
+
 def _executable_path(executable_name):
     return subprocess.check_output(['which', executable_name]).strip()
 
+
 def exec_docker(*args):
-    os.execl(_executable_path('docker'), 'docker', *args)
+    updated_env = copy(os.environ)
+    updated_env.update(get_docker_env())
+    args = args + (updated_env,)
+    os.execle(_executable_path('docker'), 'docker', *args)
```

Comment:
it used to be an `os.execl` command which runs the executable and causes the new process to inherit the environment of the of the current process. `os.execle ` tells it that we will be providing the environment. Original I had just provided the docker_envs, but I wanted to preserve current behavior but augment it with the docker_env vars